week-1/F-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var greetingStart="hi,i'm called ";
+var name="daniella";
+var greeting=greetingStart + name;
+console.log(greeting);
```

Comment:
It's a good practice to leave a space after the equal sign i.e. greetingStart = "hi, i'm called ";

Reply:
notes taken.thanks